app/src/main/res/layout/item_card_order.xml

```diff
@@ -82,15 +82,25 @@
                         android:layout_height="wrap_content"
                         android:maxLines="1"
                         android:ellipsize="end"
-                        android:layout_marginBottom="@dimen/layout_margin_large"
+                        android:layout_marginBottom="@dimen/layout_margin_small"
                         tools:text="Starts at 05:29 AM IST" />
 
+                    <TextView
+                        android:id="@+id/ticketsNumber"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:maxLines="1"
+                        android:ellipsize="end"
+                        android:textColor="@color/colorPrimary"
+                        android:layout_marginBottom="@dimen/layout_margin_large"
+                        tools:text="See 5 tickets" />
+
                 </LinearLayout>
 
                     <ImageView
                         android:id="@+id/eventImage"
                         android:layout_width="0dp"
-                        android:layout_height="100dp"
+                        android:layout_height="125dp"
```

Comment:
dont change the dimensions of the image. It doesn't look like a square

Reply:
I had to change it so that if the name of the event is long enough to make two lines, the "See my Tickets line" will vanish

Reply:
this is not how it is done in eventbrite right ?